worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -333,12 +346,18 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
           logDebug(s"location $location set disk hint to ${location.getStorageInfo} ")
           return fileWriter
         } catch {
+          case fe: FileAlreadyExistsException =>
+            logError("Failed to create fileWriter because of existed file", fe)
           case t: Throwable =>
             logError(
-              s"Create FileWriter for ${file.getAbsolutePath} of mount $mountPoint failed, report to DeviceMonitor",
+              s"Create FileWriter for ${file.getAbsolutePath} of mount $mountPoint " +
+                s"failed, report to DeviceMonitor",
               t)
             exception = new IOException(t)
-            deviceMonitor.reportDeviceError(mountPoint, exception, DiskStatus.READ_OR_WRITE_FAILURE)
+            deviceMonitor.reportNonCriticalError(
+              mountPoint,
+              exception,
+              DiskStatus.READ_OR_WRITE_FAILURE)
```

Comment:
Add a new type .....

Reply:
I think currently this type is sufficient to describe the possible issues, `ObservedDevice` and `Flusher` also use this type to present all unexpected IOException may meet , if we add a new type will it a bit redundant?